recipes/qubekit/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "qubekit" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qubekit/QUBEKit/archive/v.{{ version }}.tar.gz
+  sha256: 61b940d34e74315da58fed624a134a4d39454c2cf21059b01b00f2aa71c0fed8
+
+
+build:
+  entry_points:
+    - QUBEKit = QUBEKit.run:main
+    - qubekit = QUBEKit.run:main
+    - QUBEKit-pro = QUBEKit.proteins.protein_run:main
+    - QUBEKit-gui = QUBEKit.GUI.gui:main
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+    - numpy >=1.18.1
+    - scipy
+    - rdkit
+    - geometric
+    - torsiondrive
+    - ambertools
+    - networkx >=2.4
+    - matplotlib-base
+    - qcengine >=0.14.0
+
+test:
+  imports:
+    - QUBEKit
+  requires:
+    - rdkit
+  commands:
+    - QUBEKit -h
+
+about:
+  home: https://github.com/qubekit/QUBEKit
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Quantum mechanical bespoke force field derivation toolkit
+  doc_url: https://github.com/qubekit/QUBEKit
+  description: QUBEKit is a Python based force field parameter derivation toolkit. Our aims are to allow users to quickly derive molecular mechanics parameters directly from quantum mechanical calculations.
+  dev_url: https://github.com/qubekit/QUBEKit
+
+extra:
+  recipe-maintainers:
+    - jthorton
+    - CRingrose94
```

Comment:
@CRingrose94 are you OK with being a maintainer

Reply:
see above ^
> > @CRingrose94 please comment here to confirm you would like to be a maintainer as well.
> 
> I would like to be a maintainer too!

Reply:
@scopatz Yes I'd like to be a maintainer please.